### PR TITLE
Repels Implementation

### PR DESCRIPTION
**Pull request by FedericoRossiIT**
Implemented repels to prevent double battle from happening.

Added locales for repel items and description.

### CODE REVIEW

**Comment by LeCehlou** [comment]
For the french locale, it should be : "Empêche les combats doubles pendant xTurns"

**Comment by Adri1** [comment]
I confirm what LeCehlou says

**Comment by LeCehlou** [comment]
Also, if i'm not mistaken your items doesn't prevent double battles it just divide the chances per 2 ? If that's the case, i'm not sure about the description of the item. Feel free to correct me if i'm wrong

**Comment by marcnetts** [comment]
Do you really want that added into the item pool? Seems bloat, imo.

**Comment by Adri1** [comment]
One tiny edit to French :

- Current :
Empêche les combats doubles pendant {{battleCount}}

- Corrected :
Empêche les combats doubles pendant {{battleCount}} combats

**Comment by FedericoRossiIT** [comment]
> For the french locale, it should be : "Empêche les combats doubles pendant xTurns". The sentence seems a bit odd and doesn't follow the other messages that don't start with the infinitive form.

I changed the French locale. I don't really know French so I used Google Translate. Thanks for correcting the sentence.


> Also, if i'm not mistaken your items doesn't prevent double battles it just divide the chances per 2 ? If that's the case, i'm not sure about the description of the item. Feel free to correct me if i'm wrong

You were right, I was doing a dumb check since at fist I wasn't shure how it worked and never changed the `apply` method. Thank you for noticing.

> One tiny edit to French :
> 
>     * Current :
>       Empêche les combats doubles pendant {{battleCount}}
> 
>     * Corrected :
>       Empêche les combats doubles pendant {{battleCount}} combats

I'll do it right now, I didn't really knew what it said so I just copied the above thanks.

File: src/locales/de/modifier-type.ts [hunk 1]
```diff
@@ -51,6 +51,9 @@ export const modifierType: ModifierTypeTranslationEntries = {
     "DoubleBattleChanceBoosterModifierType": {
       description: "Verdoppelt die Wahrscheinlichkeit, dass die nächsten {{battleCount}} Begegnungen mit wilden Pokémon ein Doppelkampf sind.",
     },
+    "DoubleBattlePreventionModifierType": {
+      description: "Erschweren Sie die Teilnahme an zwei Schlachten pro {{battleCount}} Schlacht",
```
Comment (CodeTappert) [inline src/locales/de/modifier-type.ts:55]: `description: "Verhindere Doppelkämpfe für {{battleCount}} Kämpfe",`

Comment (CodeTappert) [reply src/locales/de/modifier-type.ts:55]: Please change this in german. Your sentence makes 0 sense

**Pull request closed without merging**